packages/angular_devkit/schematics/tasks/node-package/install-task.ts

```diff
@@ -30,6 +32,9 @@ export class NodePackageInstallTask implements TaskConfigurationGenerator<NodePa
       if (options.quiet != undefined) {
         this.quiet = options.quiet;
       }
+      if (options.hideOutput != undefined) {
```

Comment:
The below seems redundant as `hideOutput `  is declared with a type of boolean.

```ts
  if (options.hideOutput != undefined) {
        this.hideOutput = options.hideOutput;
   }
```

Reply:
It is a `boolean` in `NodePackageInstallTaskOptions`, but `options` is `Partial<NodePackageInstallTaskOptions>` so it might not be there.